tests/cassettes/test_introspect_token.yaml

```diff
@@ -0,0 +1,29 @@
+interactions:
+- request:
+    body: '{"token": "ntn_..."}'
+    headers:
+      accept:
+      - '*/*'
+      accept-encoding:
+      - gzip, deflate
+      authorization:
+      - Basic '"$BASE64_ENCODED_ID_AND_SECRET"'
```

Comment:
Hmm, how is it possible that the cassette contains this Authorization header if we didn't implement it? Do the endpoint also work with a normal token, and then you changed the cassette manually? (We should never modify the cassettes manually.)

Reply:
I did modify the cassette 😅. When I ran the test originally, I hard-coded the authorization because of the difference mentioned in #260. Even with those changes fixed, won't the authorization and token not be obfuscated in the cassette, or if set correctly they will?